### DISCUSSION TITLE
Are there any plans for adding LSP support in CotEditor?

### DISCUSSION THREAD

**Discussion by ghost** [coteditor/CotEditor#1594] [Q&A] [answered]
As the title suggest, I'm wondering if there any plans for adding LSP support in CotEditor, it's a functionality expected from any modern code editor from lightest to heaviest..

**Comment by ghost** [comment]
@1024jp ?

**Comment by 1024jp** [comment, marked as answer]
The short answer is: "Currently not".
The priority is not high for me.

cf. #1524
